components/api/flow.js

```diff
@@ -35,6 +39,10 @@ const Flow = ({ url }) => (
     <SectionLayout sub title="Installing the definitions">
       <CodeBlock code={install} />
     </SectionLayout>
+  
+    <p>If you are on npm >= 5.2 you could simply use npx</p>
```

Comment:
Maybe we should use the `<Note>` component for this and just add a link to the `npx` repo?